crates/mun_codegen/src/ir/type_table.rs

```diff
@@ -136,7 +142,10 @@ impl<'a, D: IrDatabase> TypeTableBuilder<'a, D> {
     /// Collects unique `TypeInfo` from the specified function signature and body.
     pub fn collect_fn(&mut self, hir_fn: hir::Function) {
         // Collect type info for exposed function
-        if !hir_fn.data(self.db).visibility().is_private() || hir_fn.is_extern(self.db) {
+        if !hir_fn.data(self.db).visibility().is_private()
```

Comment:
Are some of these options obsolete now? As in, if a function is externally visible is it not by definition in the dispatch table?

Reply:
Yup. You are right. I'll fix that.